frontend/packages/console-plugin-sdk/src/typings/nav.ts

```diff
@@ -10,7 +10,8 @@ import {
 
 namespace ExtensionProperties {
   interface NavItem {
-    section: NavSectionTitle;
+    perspective?: string;
+    section?: NavSectionTitle | string;
```

Comment:
The inferred type becomes `string`, so I'd just use `section?: string;` here.

`NavSectionTitle` is now admin-perspective related, but `NavItem` should cover any new perspectives.

Reply:
I have removed `NavSectionTitle`